tests/sentry/grouping/snapshots/test_grouphash_metadata/test_metadata_from_variants/newstyle@2019_05_08/unreal_ensure_check_fail_on_windows.pysnap

```diff
@@ -0,0 +1,202 @@
+---
+created: '2025-02-19T15:23:51.092540+00:00'
+creator: sentry
+source: tests/sentry/grouping/test_grouphash_metadata.py
+---
+hash_basis: stacktrace
+hashing_metadata: {
+  "num_stacktraces": 1,
+  "stacktrace_location": "exception",
+  "stacktrace_type": "system"
+}
+---
+metrics with tags: {
+  "grouping.grouphashmetadata.event_hash_basis": {
+    "hash_basis": "stacktrace",
+    "is_hybrid_fingerprint": "False"
+  },
+  "grouping.grouphashmetadata.event_hashing_metadata.stacktrace": {
+    "chained_exception": "False",
+    "stacktrace_location": "exception",
+    "stacktrace_type": "system"
+  }
+}
+---
+contributing variants:
+  system*
+    hash: "e7fcadedd547b7ab9677f58d2547d479"
+    contributing component: exception
+    component:
+      system*
+        exception*
+          stacktrace*
+            frame*
+              function*
+                "RtlUserThreadStart"
```

Comment:
In this file, this frame is shown at the top.
In the UI ([link](https://ivan-tustanivskyi.sentry.io/issues/6233924885/events/eb2737e5c27145405874d398f0485582/?project=4508533445754880) to issue) shows at the bottom:
<img width="712" alt="image" src="https://github.com/user-attachments/assets/581c9c6b-24c4-46c3-acf0-4194fddbec42" />

In the grouping config section matches what we see in this file (at the top):
<img width="716" alt="image" src="https://github.com/user-attachments/assets/14dae34f-8e4c-45d8-81ee-fc40b0720494" />